README.md

```diff
@@ -12,32 +15,59 @@ Thoth is deployed to the following [environments](docs/environments.md).
+
+- Bots also allow user to configure them to auto update the deployment of the component in thoth-application. For this firstly, we have to setup a component for the application.
+  The component must be created based on the description of the project. Following the kustomize methods the component should have two sub-directory **base** and **overlays**, which is required by argocd applications for deployment.
+  Base would contain files which would be deployed across all environments(Prod/Stage/Test), be sure to add all relevant files. (Note: If you feel a file configuration seems like a environment specific thing then it definitely doesn't belong in the base ).
+  Overlays would contains more sub-directories based on environments (Prod/Stage/Test), please follow the pattern of other component. Add the relevant files to the overlays, you are working on.
+
+- Once the component contains all information requirement, we have to add an argo-cd application configuration for the component to be supported by argocd.
+  For Stage and Test: Please make an Pull request to [aicoe-cd](https://github.com/AICoE/aicoe-cd) , Similar to this [PR](https://github.com/AICoE/aicoe-cd/pull/136/files)
+  For Production: Please make a Pull Request to [argo-apps](https://github.com/operate-first/argocd-apps), Similar to this [PR](https://github.com/operate-first/argocd-apps/pull/20)
+
+NOTE: Production environment is to be updated by aicoe-thoth-devops Members only.
+
 #### Component secrets
+
 We use [sops](https://github.com/mozilla/sops#sops-secrets-operations) for encrypting secrets.
 For encrypting the secrets, the contributor can do the following:
- - First, create a `secrets.yaml` at the root level. (git would ignore the file)
- - Then run sops command to encrypt them to the desired application subdirectory.
-  Example: `sops -e secrets.yaml > {component}/overlays/test/secrets.enc.yaml`
- - Make sure to contribute the encrypted **secrets.enc.yaml** .
+
+- First, create a `secrets.yaml` at the root level. (git would ignore the file)
+- Then run sops command to encrypt them to the desired application subdirectory. Example: `sops -e secrets.yaml > {component}/overlays/test/secrets.enc.yaml`
+- Make sure to contribute the encrypted **secrets.enc.yaml** .
 
 ## Open Questions
 
-### how do we tag release
+- What should i do if i don't know how to deploy my application?
+
+  we all are happy to help, open an issue with details of the application.
+  Team would guide and help you with all the rest
+
+- How do we tag release ?
+
+  Make use of the Thoth-Bots Kebechet and AICoE-CI , which helps application to release as container image or to pypi based on the nature of the application.
+
+- How do we know application is deployed in Thoth Component and running ?
```

Comment:
`How do we know application is deployed in Thoth Component and running ? not sure what is Thoth Component in the question, should be:

`How do we know Thoth component application is deployed in cluster and running ?